titanic.py

```diff
@@ -73,6 +73,17 @@ def conditional_mutual_information(p, j, *conditional_indices):
     return (conditional_shannon_entropy(np.sum(p, axis=j), *conditional_indices)
             - conditional_shannon_entropy(p, j, *conditional_indices))
 
+def maximum_energy_delta(bqm):
```

Comment:
This function might make sense to include as a tool in Ocean

Reply:
If this finds its way to Ocean, it should be generalized to handle ising bqms (`delta *= 2` after summations)

Reply:
Also, can be made much more performant by cythonizing it.